app/src/main/kotlin/uk/govuk/app/navigation/AppLaunchNavigation.kt

```diff
@@ -32,6 +32,10 @@ internal class AppLaunchNavigation(
         if (uiState.shouldDisplayAnalyticsConsent) {
             routeStack.push(ANALYTICS_GRAPH_ROUTE)
         }
+
+        if (uiState.shouldDisplayAppUnavailable) {
```

Comment:
As App Unavailable has no further in-app navigation after it I have wondered if perhaps we shouldn't push the other graph routes in this scenario or maybe it doesn't matter and is simpler to leave as-is?

Reply:
I've had to re-jig this a bit anyway due to the bug we found during post amigo so I'll sort it out post merge